tendrl/monitoring_integration/graphite/graphite_utils.py

```diff
@@ -1,12 +1,38 @@
 import ConfigParser
 import copy
+import grp
 import os
+import pwd
 import shutil
 
 from tendrl.commons.utils import log_utils as logger
 from tendrl.monitoring_integration.grafana import constants
 
 
+def change_owner(path, uname, gname, recursive=False):
+    try:
+        uid = pwd.getpwnam(uname).pw_uid
+        gid = grp.getgrnam(gname).gr_gid
+        _chown(path, uid, gid, recursive)
+    except KeyError as ex:
```

Comment:
I would rather expect function `change_onwer` to raise the exception and not handle here. The caller should handle it.